ceph.spec.in

```diff
@@ -1438,6 +1440,11 @@ fi
 if [ $1 -eq 1 ] ; then
 /usr/bin/systemctl start ceph-osd.target >/dev/null 2>&1 || :
 fi
+%if 0%{?sysctl_apply}
+    %sysctl_apply 90-ceph-osd.conf
+%else
+    /usr/lib/systemd/systemed-sysctl %{_sysctldir}/90-ceph-osd.conf > /dev/null 2>&1 || :
```

Comment:
typo - should by "systemd-sysctl" instead of "systemed-sysctl"

Reply:
@smithfarm  yeah,you are right,I have check the code in the server,I will modify it right.Thank you very much.